hopp/simulation/technologies/battery/battery.py

```diff
@@ -162,14 +179,23 @@ def __attrs_post_init__(self):
 
         logger.info("Initialized battery with parameters and state {}".format(self._system_model.export()))
 
+
+
     def setup_system_model(self):
         """Executes Stateful Battery setup"""
-        self._system_model.setup()
+        if self.config.system_model_source == "pysam":
+            self._system_model.setup()
+        else:
+            pass
```

Comment:
You don't strictly need this if/then logic given the LDES system model has a `setup()` method that is just a `pass`

Reply:
Removed

Reply:
I've removed this logic